packages/ingest/test/test-http.js

```diff
@@ -60,7 +61,20 @@ test('Download remote file to s3 with correct content-type', async (t) => {
 });
 
 test('returns files with provider path', async (t) => {
+  const stubLink = 'file.txt ';
+  setupCrawler(stubLink);
+
   const actualFiles = await myTestHttpDiscoveryClass.list();
   const expectedFiles = [{ name: stubLink, path: myTestHttpDiscoveryClass.path }];
   t.deepEqual(actualFiles, expectedFiles);
 });
+
+
+test('strips trailing spaces from name', async (t) => {
+  const stubLink = ' fileWithSpaces.txt ';
```

Comment:
Is there any reason to include a leading space in the test? Only the trailing space will get strippied. I guess it doesn't make a difference, but I was wondering if there was a specific reason for doing that

Reply:
I wanted to make it explicit that we don't trim that bit. Maybe that's bad test design, I suppose I'm testing two things in the same test.  If we want to ensure that the leading space isn't trimmed, that should be a separate test.